ballot/admin/ballot.py

```diff
@@ -2,4 +2,9 @@
 
 from ..models import ballot as models
 
-admin.site.register(models.Ballot)
+
+class BallotAdmin(admin.ModelAdmin):
+    exclude = ['locality']
```

Comment:
I guess it was another zipcode list that took a long time.  I will take it out.  Why is a ballot locality a zipcod anyway?  Shouldn't it be a city or some other jurisdiction?